sdks/java/core/src/main/java/org/apache/beam/sdk/io/FileSystems.java

```diff
@@ -78,37 +79,25 @@ private static void loadFileSystemRegistrars() {
   }
 
   /**
-   * Sets the default configuration to be used with a {@link FileSystemRegistrar} for the provided
-   * {@code scheme}.
+   * Sets the default configuration in workers.
    *
-   * <p>Syntax: <pre>scheme = alpha *( alpha | digit | "+" | "-" | "." )</pre>
-   * Upper case letters are treated as the same as lower case letters.
+   * <p>It will be used in {@link FileSystemRegistrar FileSystemRegistrars} for all schemes.
    */
-  public static void setDefaultConfig(String scheme, PipelineOptions options) {
-    String lowerCaseScheme = checkNotNull(scheme, "scheme").toLowerCase();
-    checkArgument(
-        URI_SCHEME_PATTERN.matcher(lowerCaseScheme).matches(),
-        String.format("Scheme: [%s] doesn't match URI syntax: %s",
-            lowerCaseScheme, URI_SCHEME_PATTERN.pattern()));
-    checkArgument(
-        SCHEME_TO_REGISTRAR.containsKey(lowerCaseScheme),
-        String.format("No FileSystemRegistrar found for scheme: [%s].", lowerCaseScheme));
-    SCHEME_TO_DEFAULT_CONFIG.put(lowerCaseScheme, checkNotNull(options, "options"));
-  }
-
-  @VisibleForTesting
-  static PipelineOptions getDefaultConfig(String scheme) {
-    return SCHEME_TO_DEFAULT_CONFIG.get(scheme.toLowerCase());
+  public static void setDefaultConfigInWorkers(PipelineOptions options) {
+    defaultConfig = checkNotNull(options, "options");
   }
 
   /**
    * Internal method to get {@link FileSystem} for {@code spec}.
    */
   @VisibleForTesting
   static FileSystem getFileSystemInternal(URI uri) {
+    checkNotNull(
```

Comment:
nit: `checkState(defaultConfig != null` to throw the proper exception type. But, I can fix this in the merge if LGTY.